.dockerignore

```diff
@@ -0,0 +1,8 @@
+# application data
+/var
+# rust compiled stuff
+target
+# node modules
+node_modules
+
+.git
```

Comment:
We may be better off mirroring `.gitignore` here (similar to `.prettierignore` and `.markdownlintignore`):
1) Ensures that Docker images built for testing locally don't rely on things which aren't available in CD
2) Ensures that locally-built images don't accidentally include `.env.local` files (or other secret things not on this list)